python/pyproject.toml

```diff
@@ -19,7 +19,9 @@
 requires = [
     "cython >= 0.29",
     "numpy==1.14.5; python_version<'3.7'",
-    "numpy==1.16.0; python_version>='3.7'",
```

Comment:
Files:

- python/pyproject.toml
- python/requirements-wheel-build.txt
- python/requirements-wheel-test.txt

Seemed to closely match, but `pyproject.toml` mentions a different version for Python 3.7, I pinned what was specified for Python 3.8, but it probably can accept `1.19.2` which I'm not sure if it's the first release to support Python 3.9, but it does build properly.

Reply:
> but it probably can accept 1.19.2 which I'm not sure if it's the first release to support Python 3.9

How about confirming it with CI?

Reply:
Just to follow up, 1.19.2 did work for me, I'm just saying there might be a earlier release that works. It's probably simplest/best to use 1.19.2 since it's mentioned in their changelog:

> This release supports Python 3.6-3.8. Cython >= 0.29.21 needs to be used when building with Python 3.9 for testing purposes.
https://numpy.org/devdocs/release/1.19.2-notes.html

...looking at 1.19.1 it says the same thing so it might be safe to go backwards